docker/nginx.site.conf

```diff
@@ -7,11 +7,17 @@ server {
 
     # location = /favicon.ico { access_log off; log_not_found off; }
     location /sitestatic/ {
-        root $PROJECT_PATH;
+        root /app;
     }
 
     location / {
         include /etc/nginx/proxy_params;
-        proxy_pass http://unix:${PROJECT_CONF}/gunicorn.sock;
+        proxy_pass http://unix:/app-conf/gunicorn.sock;
```

Comment:
It's good use the environment variable again. The Nginx does not accept variables like that, but before configure on nginx directory, this file is processed to set the values.

The lines that is interesting pay attention is: 10, 15 and 18.

This line that set the values on Dockerfile:
https://github.com/Ilhasoft/rapidpro/blob/c390161fef9788cdd0c816d115ce238513b16705/docker/Dockerfile#L44